src/3rdparty/walkontable/src/cell/range.js

```diff
@@ -80,27 +83,52 @@ class CellRange {
    * @returns {boolean}
    */
   isSingle() {
-    return this.from.row === this.to.row && this.from.col === this.to.col;
+    return this.from.row >= 0 && this.from.row === this.to.row &&
```

Comment:
It will return `true` for selection when there is selected only column header and all rows are trimmed. Is it okay?

Reply:
There will be only `true` when a cell is selected. If the selection was made by header click it always returns `false`. I think that it is OK.